index.d.ts

```diff
@@ -164,9 +164,9 @@ export interface Options extends RequestInit {
 	/**
 	Shortcut for sending JSON. Use this instead of the `body` option.
 
-	Accepts a plain object which will be `JSON.stringify()`'d and the correct header will be set for you.
+	Accepts any plain object or value, which will be `JSON.stringify()`'d and sent in the body with the correct header set.
 	*/
-	json?: unknown;
+	json?: object | Array | string | number | boolean | null | undefined;
```

Comment:
undefined is unnecessary here

Reply:
that's fine... but what's your reasoning?  `undefined` is a valid value to pass the json prop here... that said, this is my first time working with ts so I'm not aware of the subtleties...